### ISSUE TITLE
Remove the /15.0 throughput normaliser; report raw items/second

### ISSUE THREAD

**Issue by beyarkay** [beyarkay/factorion#120]
## What

Two call-sites divide raw throughput (in items/second) by 15.0 to convert into "fraction of a Belt-1's max flow":

- `factorion.py:1077` — `eval_model`:
  ```python
  throughput = torch.tensor(
      funge_throughput(normalised_world)[0] / 15.0,
      dtype=value.dtype,
  )
  ```
- `ppo.py:512` — `FactorioEnv.step`:
  ```python
  throughput, num_unreachable = factorion_rs.simulate_throughput(...)
  # TODO don't always divide by 15
  throughput /= 15.0
  ```

The TODO already flags the issue in `ppo.py`.

## Why remove it

- `/ 15.0` is only correct when a factory's bottleneck is a single Belt-1. Splitters max out at 30/s, multi-lane factories scale linearly beyond that; the "normalised" value is then >1.0 and stops meaning what its name implies.
- Logging raw items/second is comparable across factory sizes, lesson kinds, and recipe choices, without needing to remember the magic constant.
- W&B charts of throughput (`val/throughput`, `val/<kind>/throughput`) currently inherit the divided value via the env's `info['throughput']`; switching to raw items/second makes them directly readable.

## What to change

1. Drop the `/15.0` in `ppo.py:512` and `factorion.py:1077`.
2. Audit reward-shaping coefficients (`Args.coeff_throughput = 0.8785` in `ppo.py:107`) — they were tuned against the normalised value, so the magnitudes will need to be rescaled (or the coefficient docstring updated to reflect the new units).
3. Update any tests that assert on throughput ranges (e.g. `tests/test_lesson_generation.py` already uses raw items/second for its 30.0 splitter cap, so those should be unaffected).

## Not fixing now

Filed for visibility.
